app/scripts/modules/core/src/artifact/react/ExpectedArtifactSourceSelector.tsx

```diff
@@ -0,0 +1,72 @@
+import { module } from 'angular';
+import * as React from 'react';
+import { react2angular } from 'react2angular';
+import { UUIDGenerator, TetheredSelect, IExpectedArtifact } from 'core';
+
+export interface IExpectedArtifactSourceOption {
+  source: {
+    expectedArtifacts: IExpectedArtifact[];
+    name: string;
+  };
+  value?: string;
+  label: string;
+}
+
+export interface IExpectedArtifactSourceSelectorProps {
+  sources: IExpectedArtifactSourceOption[];
+  selected: IExpectedArtifactSourceOption;
+  onChange: (_: IExpectedArtifactSourceOption) => void;
```

Comment:
`_` implies to me that the parameter is unused

Reply:
ah yeah fair point. mostly just didn't want to have to think of a name for a type annotation where's it's otherwise unused.